docs/contribute/contribute.md

```diff
@@ -76,17 +76,19 @@ We've installed the development version with [Valet](https://laravel.com/docs/va
 
 Once the above softwares are installed (or if you've finished the installation of Homestead/Valet):
 
-1. `composer install` in the folder the repository has been cloned.
+1. `make build-dev` in the folder the repository has been cloned. This will run :
+    - `composer install` to install all packages.
+    - `npm install` to install all the front-end dependencies and tools needed to compile assets.
+    - `npm run dev` to compile js and css assets.
 1. `cp .env.example .env` to create your own version of all the environment variables needed for the project to work.
```

Comment:
It would be good to do `cp .env.example.env` and `php artisan key:generate` in the makefile as well.

Reply:
There is a `make install` command for that ;) Except for the `cp .env.example .env` command. I could add it if you insist :)